hooks/boot/02-message-helpers.rb

```diff
@@ -38,11 +38,13 @@ def dev_new_install_message(kafo)
   end
 
   def failure_message
-    say "  <%= color('Something went wrong!', :bad) %> Check the log for ERROR-level output"
+    say "\n  <%= color('There were errors detected during install.', :bad) %>"
+    say "  Please address the errors and re-run the installer to ensure the system is properly configured."
```

Comment:
```suggestion
    say "\n  There were <%= color('ERROR', :bad) %>s detected during install."
    say "  Please address the <%= color('ERROR', :bad) %>s and re-run the installer to ensure the system is properly configured."
```

I want to make this scream "hey you! you need to do something about this!" as much as possible. What do you think @ehelms ?